lib/Controller/FederatedInvitesController.php

```diff
@@ -239,9 +239,19 @@ public function inviteAccepted(string $token = '', string $provider = ''): JSONR
 			$recipientProvider = $this->federatedInvitesService->getProviderFQDN();
 			$client = $this->httpClient->newClient();
 			$responseData = null;
+			$provider_url = parse_url($provider);
+			$scheme = $provider_url['scheme'] ?? 'https';
+			$domain = $provider_url['host'] ?? '';
+			if ($domain == '') {
+				$this->logger->error("Invalid provider URL: $provider", ['app' => Application::APP_ID]);
+				return new JSONResponse(['message' => 'Invalid provider URL.'], Http::STATUS_NOT_FOUND);
+			}
+			if (isset($provider_url['port'])) {
```

Comment:
Do we want to allow ports, is that needed? If provider would be a FQDN (by future spec) than that does not contain a port.